### PR TITLE
combatiblity for gnome-shell 3.32 (maybe break for earlier versions)

### LINKED ISSUE #47: TeaTime stopped working on gnome-shell 3.32
Many gnome shell apps, including TeaTime, stopped working under gnome-shell 3.32.  There is something the 3.32 upgrade requiring ES6 standards.  

Here is a change in the Arc-Menu project that addresses it:
https://gitlab.com/LinxGem33/Arc-Menu/commit/e64ff70ea2d5699c31c9245f06bbb09ef09b2d2d

There is a note about backwards compatibility issues for systems that have gnome-shell < 3.32 in Arc-Menu's readme: https://gitlab.com/LinxGem33/Arc-Menu/blob/master/README.md

### PR DESCRIPTION
**Pull request by zeners**
fixed #47 but may be break earlier versions

### CODE REVIEW

**Comment by zeners** [comment]
also fixed parsing of time:
only minutes, only seconds with : prefixed:
120 => 2:00:00
:20 => 00:20

**Comment by bike-bill** [comment]
Thanks @zeners.  This PR is working nicely!

**Comment by sanjayankur31** [comment]
The extension doesn't work when installed from extensions.gnome.org. Could this fix please be merged and pushed there?

File: src/icon.js [hunk 1]
```diff
@@ -10,86 +10,90 @@
...
-
-		cr.translate(padding_x, padding_y);
-		cr.scale(scaling, scaling);
-
-		this._drawingObject.draw(cr, this._customStatus, this._primaryColor, this._secundaryColor);
-
-		cr.restore();
+		this.myinit = function () {
+			this._base_size = size;
+			//this.setScaling(Utils.getGlobalDisplayScaleFactor());
```
Comment (oleid) [inline src/icon.js]: Any specific reason to comment this?

Comment (zeners) [reply src/icon.js]: not at this time, currently the re-uncommented line works, would you mind to correct it by yourself or should i create a new commit (there are others based on this commit)?
// sry for delay, missed you comment on code

**Pull request merged**